src/childHealth/config.py

```
@@ -0,0 +1,28 @@
+    cat_features: List[str]
+    target: str
+
+    @classmethod
+    def from_yaml(cls, config_path: str) -> 'ProjectConfig':
+        """Load configuration from a YAML file."""
+        try:
+            with open(config_path, 'r') as f:
+                config_dict = yaml.safe_load(f)
+            return cls(**config_dict)
+        except FileNotFoundError:
+            raise FileNotFoundError(f"Configuration file not found: {config_path}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)